### PR TITLE
feat: add information about row/column layout for radio and checkboxes

### PR DESCRIPTION
**Pull request by haakemon**
This feature was added in https://github.com/Altinn/altinn-studio/pull/8238

### CODE REVIEW

File: schemas/json/layout/layout.schema.v1.json [hunk 1]
```diff
@@ -399,7 +399,7 @@
         "enum": ["validation", "validatePage", "validateAllPages", "calculatePageOrder"]
       }
     },
-    "selectionComponents": {
+    "dropdownComponents": {
```
Comment (lorang92) [inline schemas/json/layout/layout.schema.v1.json:402]:
```suggestion
    "selectionComponents": {
```

File: schemas/json/layout/layout.schema.v1.json [hunk 2]
```diff
@@ -115,16 +115,16 @@
       "allOf": [
         { "if": {"properties": {"type": { "const": "AddressComponent"}}}, "then": { "$ref": "#/definitions/addressComponent"}},
         { "if": {"properties": {"type": { "const": "AttachmentList"}}}, "then": { "$ref": "#/definitions/attachmentListComponent"}},
-        { "if": {"properties": {"type": { "const": "Checkboxes"}}}, "then": { "$ref": "#/definitions/selectionComponents"}},
+        { "if": {"properties": {"type": { "const": "Checkboxes"}}}, "then": { "$ref": "#/definitions/radioAndCheckboxComponents"}},
         { "if": {"properties": {"type": { "const": "Datepicker"}}}, "then": { "$ref": "#/definitions/datepickerComponent"}},
-        { "if": {"properties": {"type": { "const": "Dropdown"}}}, "then": { "$ref": "#/definitions/selectionComponents"}},
+        { "if": {"properties": {"type": { "const": "Dropdown"}}}, "then": { "$ref": "#/definitions/dropdownComponents"}},
```
Comment (lorang92) [inline schemas/json/layout/layout.schema.v1.json:120]:
```suggestion
        { "if": {"properties": {"type": { "const": "Dropdown"}}}, "then": { "$ref": "#/definitions/selectionComponents"}},
```

File: schemas/json/layout/layout.schema.v1.json [hunk 3]
```diff
@@ -427,6 +427,40 @@
         }
       }
     },
+    "radioAndCheckboxComponents": {
```
Comment (lorang92) [inline schemas/json/layout/layout.schema.v1.json:430]: Intead of a `radioAndCheckboxComponents` where we duplicate all properties in `selectionComponents` I think we should "extend" the selectionComponents using `allOf`. Something like this:
```
"radioAndCheckboxComponents": {
      "allOf": [{ "$ref": "#/definitions/selectionComponents" }],
      "properties": {
        "layout": {
          "type": "string",
          "enum": ["column", "row"],
          "title": "Layout",
          "description": "Define the layout style for the options"
        }
      }

**Review by lorang92 — CHANGES_REQUESTED** [review]

File: schemas/json/layout/layout.schema.v1.json [hunk 4]
```diff
@@ -427,6 +427,40 @@
...
+          "description": "Boolean value indicating if the options should be instance aware. Defaults to false. See more on docs: https://docs.altinn.studio/app/development/data/options/"
+        },
+        "layout": {
+          "type": "string",
+          "enum": ["column", "row"],
+          "title": "Layout",
+          "description": "Define the layout style for the options"
+        }
+      }
+    },
```
Comment (lorang92) [inline schemas/json/layout/layout.schema.v1.json:463]:
```suggestion
      "allOf": [{ "$ref": "#/definitions/selectionComponents" }],
      "properties": {
        "layout": {
          "type": "string",
          "enum": ["column", "row"],
          "title": "Layout",
          "description": "Define the layout style for the options"
        }
      }
    },
```

File: schemas/json/layout/layout.schema.v1.json (hunk 3 above)
Comment (haakemon) [reply schemas/json/layout/layout.schema.v1.json:430]: Nice, that is a lot better!

**Review by lorang92 — APPROVED** [review]
Nice! Approved.  But I think @altinnadmin or @SandGrainOne has to merge

**Pull request merged**